O21.Game/U95/Sprites.fs

```diff
@@ -60,13 +62,36 @@ module Sprites =
         Array.init backgroundGraphics.Length (fun i ->
             createSprite device backgroundGraphics[i] backgroundGraphics[i]
         )
-         
+        
+    // TODO: calculate indexes
```

Comment:
Could you please clarify and/or implement this TODO in this PR?

Reply:
This is about the order in which the bitmaps are stored in dll file: 45 transparency masks for left direction-> 45 graphics bitmaps and same for other direction. I will Implement it in next commit.